e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -455,13 +455,192 @@ test.describe('Example imagery thumbnails resize in display layouts', () => {
-    test.fixme('If the imagery view is in pause mode, it should not be updated when new images come in');
-    test.fixme('If the imagery view is not in pause mode, it should be updated when new images come in');
+    test('Example Imagery in Flexible layout', async ({ page }) => {
+        // Go to baseURL
+        await page.goto('/', { waitUntil: 'networkidle' });
+
+        // Click the Create button
+        await page.click('button:has-text("Create")');
+
+        // Click text=Example Imagery
+        await page.click('text=Example Imagery');
+
+        // Clear and set Image load delay (milliseconds)
+        await page.click('input[type="number"]', {clickCount: 3});
+        await page.type('input[type="number"]', "20");
+
+        // Click text=OK
+        await Promise.all([
+            page.waitForNavigation({waitUntil: 'networkidle'}),
+            page.click('text=OK'),
+            //Wait for Save Banner to appear
+            page.waitForSelector('.c-message-banner__message')
+        ]);
+        // Wait until Save Banner is gone
+        await page.waitForSelector('.c-message-banner__message', { state: 'detached'});
+        await expect(page.locator('.l-browse-bar__object-name')).toContainText('Unnamed Example Imagery');
+        const bgImageLocator = await page.locator(backgroundImageSelector);
+        await bgImageLocator.hover();
+
+        // Click the Create button
+        await page.click('button:has-text("Create")');
+
+        // Click text=Flexible Layout
+        await page.click('text=Flexible Layout');
+
+        // Assert Flexable layout
+        await expect(page.locator('.js-form-title')).toHaveText('Create a New Flexible Layout');
+
+        // Click text=OK
+        page.click('text=OK');
+
+        // Wait until Save Banner is gone
+        await page.waitForSelector('.c-message-banner__message', { state: 'detached'});
+
+        // Click My Items
+        await page.locator('form[name="mctForm"] >> text=My Items').click();
+        await Promise.all([
+            page.waitForNavigation({waitUntil: 'networkidle'})
+        ]);
+
+        // Click My Items
+        await page.locator('.c-disclosure-triangle').click();
+
+        // Right click example imagery
+        await page.click(('text=Unnamed Example Imagery'), { button: 'right' });
+
+        // Click move
+        await page.locator('.icon-move').click();
+
+        // Click triangle to open sub menu
+        await page.locator('.c-form__section .c-disclosure-triangle').click();
+
+        // Had to use Xpath here I could not click this with css. Click unnamed flexview
+        await page.locator('xpath=//div[2]/div/div[2]/div/div/form/div/div[2]/div/div[3]/div/div[2]/div[2]/div/div[3]/div/a').click();
+
+        // Click text=OK
+        await Promise.all([
+            page.click('text=OK')
+            //Wait for Save Banner to appear
+        ]);
+
+        // Save template
```

Comment:
@afahey87 since we're doing this same series of steps 3 times in this test suite, this would be a great opportunity to change this a common function. Check out the `e2e/tests/plugin/plots` folder to see some examples

Reply:
Yes, I have been wondering how functions were used. I will DRY up the test.